src/sentry/integrations/jira/integration.py

```diff
@@ -360,6 +360,8 @@ def get_group_description(self, group, event, **kwargs):
             )
         ]
 
+        # TODO: We probably need to modify something here so that we can use the occurrence to do
+        # this customization rather than the performance event.
```

Comment:
+1
I haven't added evidence_display to detectors yet. will work on this tomorrow